README.md

```diff
@@ -2,24 +2,250 @@
+
+#### Provisional acceptance
+The process for acceptance to Provisional status begins when the author(s) assign an "**RFC**" label to the GitHub PR, add an empty "**Straw Poll**" comment to it, and send an announcement of the PR to the PMIx community mailing list requesting consideration of the PR for Provisional status. The announcement shall include a link to the PR, a brief description of the proposed change, and indicate any desired timeline for approval. It is especially important that proposals tied to contractual milestones be so designated to ensure that members are alerted to potential time pressures.
+
+The "Straw Poll" comment is used by the ASC as a mechanism for obtaining a sense of how many people have viewed the PR and are following the discussion. This serves no official purpose other than to help define participation in the consensus building process and record the following tally:
+
+* Those participants that approve the proposal must indicate that by placing a :+1: emoji on the comment.
+
+* Those participants that have specific concerns regarding the proposal must indicate that by placing a :-1: emoji on the comment.
+
+Anyone indicating concern must provide a comment on the ticket following the "**Straw Poll**" comment describing their concern. Sufficient detail should be provided to allow the author(s) to respond to the specific concerns. The author(s) can then work to address the concern, modifying the PR as required - modification should be accompanied by an appropriate comment citing the original participant who raised the concern to ensure they are notified of the change. Participants shall indicate their acceptance or rejection of the modification by so marking the change comment and adjusting their original "vote" on the straw poll if appropriate.
+
+The ASC Co-Chairs shall monitor the status of discussion on the PR. Once discussion appears to have consolidated (either measured by participation in the straw poll, rate of comments, or a combination of both), the Co-Chairs will schedule the proposal for a reading at the next quarterly meeting of the ASC. The proposal shall be included in the announced agenda for the meeting along with a link to the PR and the results of the straw poll.
+
+At least one proposal author must attend the quarterly meeting to present the PR and participate in discussion regarding its approval. A "reading" of the proposal shall be conducted at the meeting with the author(s) reviewing any questions/concerns raised and how they were addressed. During the presentation, someone designated by the Co-Chairs will take notes on the PR (and/or Issue) on behalf of the presenter. At the end of the agenda item, those members wishing to adjust their straw votes will be given an opportunity to do so. Members who did not attend the meeting will also have a chance to adjust their votes based on the published minutes of the meeting.
```

Comment:
Can participants adjust their straw votes too based on the discussion or only members?

Reply:
Anyone can:

> At the end of the agenda item, anyone wishing to adjust their straw votes will be given an opportunity to do so. Participants who did not attend the meeting will also have a chance to adjust their votes based on the published minutes of the meeting.